### PR TITLE
Add caption ljohnston97

### PR DESCRIPTION
**Pull request by ljohnston97**
## Pull Request
This PR updates the caption because it was out of date.
This PR closed / resolves / fixes #2
 
## Done
- created a branch
- made a commit
 
## To do
- [ ] merge the branch
- [ ] delete the ## My Pull Request

### CODE REVIEW

File: _slides/02-Adomonir.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+My first caption
```
Comment (OmarOrion) [inline _slides/02-Adomonir.md:7]: Good job! looks great :)

**Review by OmarOrion — APPROVED** [review]
approved

File: _slides/02-Adomonir.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (Adomonir) [inline _slides/02-Adomonir.md:6]: This looks good!

Comment (Adomonir) [inline _slides/02-Adomonir.md:6]:
```suggestion
My first caption, please add more details
```

**Review by Adomonir — APPROVED** [review]
LG 😄

**Pull request merged**